src/Controller/EnMarche/Coordinator/CoordinatorCitizenProjectController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche\Coordinator;
+
+use AppBundle\Coordinator\Filter\CitizenProjectFilter;
+use AppBundle\Entity\CitizenProject;
+use AppBundle\Exception\BaseGroupException;
+use AppBundle\Form\CoordinatorAreaType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * @Route("/espace-coordinateur/projet-citoyen")
+ * @Security("has_role('ROLE_COORDINATOR_CITIZEN_PROJECT')")
+ */
+class CoordinatorCitizenProjectController extends Controller
+{
+    /**
+     * @Route(path="/list", name="app_coordinator_citizen_project")
+     */
+    public function listAction(Request $request)
```

Comment:
Missing return typehint or remove it for `validateAction()`

Reply:
done 👍